source/access/getting_access.rst

```diff
@@ -1,57 +1,213 @@
-   Community.
-Researchers at other Flemish public research institutions
-   You can get compute on the Tier-1 infrastructure through a project
-   application or access the Tier-2 infrastructure through contact with
-   one of the coordinators.
-Businesses, non-Flemish public knowledge institutions and not-for-profit organisations
-   These entities can buy compute time on the
-   infrastructure. The procedures are explained on the page
-   ":ref`buying compute time`".
+also called a VSC account.  Check the `VSC website <eligible users_>`_
+for conditions.
+
+All VSC-accounts start with the letters \\"vsc\" followed by a
+five-digit number. The first digit gives information about your home
+institution. There is no relationship with your name, nor is the
+information about the link between VSC-accounts and your name publicly
+accessible.
+
+Your VSC account gives you access to most the VSC Tier-2 infrastructure, though
+for some more specialised hardware you have to ask access separately.
+The rationale is that  we want to ensure that that (usually rather expensive)
+hardware is used efficiently for the type of applications it was purchased for.
+Contact your :ref:`local VSC coordinator<get in touch_>`_ to arrange access
+when required.
+
+For the main Tier-1 compute cluster you need to submit a
+`project application <tier-1 project application_>`_ (or you should be
+covered by a project application within your research group).
+
+
+.. _account request:
+
+How to request an accout?
+-------------------------
+
+Unlike your institute account, VSC accounts don't use regular fixed
+passwords but a key pair consisting of a public an private key because
+that is a more secure authentication technique.  To apply for a VSC
+account, you need a public/private key pair.
+
+.. _create key pair:
+
+Create a public/private key pair
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A key pair consists of a private and a public key.
+
+#. The private key is stored on the computer(s) you use to access the VSC
+   infrastructure and always stays there.
+#. The public key is stored on the  VSC systems you want to access, allowing
+   to prove your identity through the corresponding private key.
+  
+How to generate such a key pair depends on your operating system. We
+describe the generation of key pairs in the client sections for
+
+- :ref:`Linux<generating keys linux>`,
+- :ref:`Windows <generating keys windows>` and
+- :ref:`macOS<generating keys macos>` (formerly OS X).
+
+Without a key pair, you won't be able to apply for a VSC account.
+
+.. warning::
+
+   It is clear from the above that it is very important to protect your
+   private key well. Therefore:
+   
+   - You should choose a strong passphrase to protect your private key.
+   - You should not share your key pair with other users.
+   - If you have accounts at multiple supercomputer centres (or on other
+     systems that use SSH), you should seriously consider using a
```

Comment:
why not just: `... you should use a different key pair for each of those accounts` ?

Reply:
This is the original formulation, we can't enforce this anyway.  I  can emphasize it a bit more.

Reply:
you can leave it as is if you prefer, it was just a suggestion.